java.html.markdown

```diff
@@ -99,7 +99,21 @@ public class LearnJava {
         System.out.println(fooString);
         System.out.println(barString);
         System.out.println(bazString);
-
+        
+        // String Building
+        // #1
+        String plusConcatenated = "Strings can " + "be concatenated " + "via + operator";
+
+        // #2
+        StringBuilder builderConcatenated = new StringBuilder();
```

Comment:
Nah, it might be good to teach people not to abuse `+`, because of performance implications.

However, the mention of `StringBuffer` (the thread-safe version) might not be necessary